jinja/plugins.html

```diff
@@ -25,17 +25,19 @@ <h4 class="card-header juicy-peach-gradient">default.qubit</h4>
                     <div class="card-body">
                         <p class="card-text">
                             A simple state-vector qubit simulator written in Python.
-                            <br><br>A good choice for optimizations with a moderate number of qubits and parameters.
+                            <br><br>A good choice for optimizations with a moderate number of qubits and parameters, and stochastic
+                            expectation values.
                         </p>
                     </div>
                 </a>
             </div>
             <div class="card plugin-card">
                 <a href="https://pennylane.readthedocs.io/en/stable/code/api/pennylane.devices.default_qubit_tf.DefaultQubitTF.html">
-                    <h4 class="card-header juicy-peach-gradient">default.qubit.tf</h4>
+                    <h4 class="card-header juicy-peach-gradient">default.qubit.tf<br> default.qubit.autograd</h4>

```

Comment:
Also support having an extra card, it looks a bit too busy in the title area:
![image](https://user-images.githubusercontent.com/49409390/90763857-6f26e780-e2b5-11ea-8229-0127f2444b58.png)

Although yeah the layout may be tough :thinking:

Reply:
Yeah, I'm too much of a perfectionist, and couldn't work out a layout that looked nice! adding an extra row also looked quite ugly to me, and added additional vertical scrolling to the page.